### PR TITLE
Added license information for events (fix: #324)

### PR DESCRIPTION
**Pull request by haardikdharma10**
This PR solves issue #324. Added information to the `event` single template.

### CODE REVIEW

File: layouts/event/single.html [hunk 1]
```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
+            <br>{{ T "blog-licensed-under" }} <a
```
Comment (zner0L) [inline layouts/event/single.html:33]: I don't think that this text is fitting here. Something short like "License" would be much better. You can add this to the translations file in `src/i18n/en.json` in the `hugo` section.

File: layouts/event/single.html [hunk 2]
```diff
@@ -30,6 +30,11 @@
...
+                    href="{{ T (printf "lic-url-%s" ($.Scratch.Get "license")) }}">{{ T (printf "lic-%s" ($.Scratch.Get "license")) }}</a>
+                <span class="hidden" xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Dataset"
+                    property="dct:title" rel="dct:type">{{ .Title }}</span>
+                    {{ with .Params.notices }}<p>{{ . | markdownify }}</p>{{ end }}
```
Comment (zner0L) [inline layouts/event/single.html:37]: The notices won't fit in the small space under the video. Take a look at the build preview: https://deploy-preview-410--datenanfragen.netlify.app/en/verein/event/okl-berlin-05-2020/

Maybe move those someplace else.

File: layouts/event/single.html [hunk 3]
```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
```
Comment (zner0L) [inline layouts/event/single.html:32]: Please copy the format of the other information under the video. Wrap the license information in a `<div>` and have the label be `<strong>`.

**Review by zner0L — CHANGES_REQUESTED** [review]
Thank you for your contribution! There are still some things to improve. Please take a look at the comments above. Also, apparently the `okl-berlin-05-2020` event contained an error (not your fault, though). To make it work, you'll need to remove the dot from the license information.

File: layouts/event/single.html (hunk 1 above)
Comment (haardikdharma10) [reply layouts/event/single.html:33]: I think hugo section already contains a translation on line number 111:
Screenshot 2020-10-02 at 4 24 04 PM

Comment (zner0L) [reply layouts/event/single.html:33]: Yes, exactly. But this translation is meant to be placed at the end of a blog post, not as a label at the top under a video. What I was suggesting was adding a new string, e.g. "License", that would be a better label in the place where you put it :)

File: layouts/event/single.html (hunk 2 above)
Comment (haardikdharma10) [reply layouts/event/single.html]: Any suggestion where should I move them?

Comment (zner0L) [reply layouts/event/single.html]: Maybe just to the bottom of the page? That seems like an appropriate place for notices.

File: src/i18n/en.json [hunk 4]
```diff
@@ -109,6 +109,7 @@
         "blog-time-at": "at",
         "blog-and": " and ",
         "blog-licensed-under": "licensed under:",
+        "license": "license",
```
Comment (zner0L) [inline src/i18n/en.json:112]:
```suggestion
        "license": "License",
```

**Comment by haardikdharma10** [comment]
All done, let me know if this looks good @zner0L?

File: layouts/event/single.html [hunk 5]
```diff
@@ -96,3 +101,4 @@
 }
 </script>
 {{ end }}
+{{ with .Params.notices }}<p>{{ . | markdownify }}</p>
```
Comment (zner0L) [inline layouts/event/single.html:104]: Please keep the (human-readable) content within the `<article>` tag and above the comments. I am sorry if I was unclear, I was talking about the bottom of the page, not the bottom of the file.

Also, you are missing an `{{ end }}` tag here.

File: layouts/event/single.html [hunk 6]
```diff
@@ -30,6 +30,11 @@
             {{ with .Params.duration }}<div>
                 <strong>{{ T "event-duration" }}</strong><br> {{ . }}
             </div>{{ end }}
+            <div><strong>{{ T "license" }} </strong> <a
+                    href="{{ T (printf "lic-url-%s" ($.Scratch.Get "license")) }}">{{ T (printf "lic-%s" ($.Scratch.Get "license")) }}</a>
+                <span class="hidden" xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/Dataset"
+                    property="dct:title" rel="dct:type">{{ .Title }}</span>
+                     </div>{{ end }}
```
Comment (zner0L) [inline layouts/event/single.html:37]: What is this `{{ end }}` tag closing? This should lead to a build error.

```suggestion
                     </div>
```

**Review by zner0L — CHANGES_REQUESTED** [review]
Thanks for putting in the work and making changes! There are some errors you introduced that are preventing the page from building, I left you some comments that hint at what got wrong. Also, can you please remove the dot from the license in the okl-berlin event (`content/(de|en)/verein/event/okl-berlin-05-2020/index.md`)? Like I said, the translations don't render correctly otherwise.

Try building the website locally ([here's how](https://github.com/datenanfragen/website#development)) and look at how the changes look on the page to prevent these errors. ;)

**Comment by haardikdharma10** [comment]
I think everthing looks fine now @zner0L 😅

**Review by zner0L — APPROVED** [review]

**Comment by zner0L** [comment]
Thanks for you contribution!

**Pull request merged**